Source/Core/InputCommon/DynamicInputTextureManager.h

```diff
@@ -11,7 +11,10 @@
 
 namespace InputCommon
 {
-class DynamicInputTextureConfiguration;
+namespace DynamicInputTextures
+{
+class DITConfiguration;
```

Comment:
Maybe just Configuration? The namespace identifier is already very long and DIT is redundant. The prefix is useful to avoid file name conflicts but it's less useful in the source code ^^